### PR TITLE
K8S 1.15.1 config

### PR DESCRIPTION
**Pull request by ahadas**
**What this PR does / why we need it**:
This PR provides the configuration needed for k8s-1.15.1 cluster in kubevirtci.

**Special notes for your reviewer**:
This PR also provides some preparations needed for #2617 .

**Release note**:

```release-note
Enable k8s-1.15.1 lane in kubevirtci that also includes Ceph by default.
```

### CODE REVIEW

File: cluster-up/cluster/k8s-1.15.1/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Kubernetes 1.15.1 in ephemeral containers
+
+Provides a pre-deployed Kubernetes with version 1.15.1 purely in docker
+containers with qemu. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are the pushed to a registry which is exposed at
```
Comment (dhiller) [inline cluster-up/cluster/k8s-1.15.1/README.md]: Nit: spelling
> ... are the**n** pushed ...

File: tests/datavolume_test.go [hunk 2]
```diff
@@ -59,8 +59,7 @@ var _ = Describe("DataVolume Integration", func() {
 	})
 
 	runVMIAndExpectLaunch := func(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
-		By("Checking that the DataVolume has succeeded")
-		tests.WaitForSuccessfulDataVolumeImport(vmi, timeout)
+		tests.WaitForSuccessfulDataVolumeImportOfVMI(vmi, timeout)
```
Comment (dhiller) [inline tests/datavolume_test.go]: Just another nit here, not a real action item: The reader does not know directly what measure the timeout value is using. We might either use a dedicated method name to point this out (`WaitSecondsForSuccessfulDataVolumeImportOfVMI`) or maybe sth in the means of [https://golang.org/pkg/time/#ParseDuration](time.ParseDuration), i.e. "5m" to increase readability. Or we agree that we **always** use seconds.

File: cluster-up/cluster/k8s-1.15.1/README.md (hunk 1 above)
Comment (ahadas) [reply cluster-up/cluster/k8s-1.15.1/README.md]: thanks I'll address this in #2617

**Comment by ahadas** [comment]
We decided to stick with a single PR (#2617)

**Pull request closed without merging**